crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -1,23 +1,37 @@
+use indicatif::{ProgressBar, ProgressStyle};
+use jstz_rollup::{
+    deploy_ctez_contract, rollup::make_installer, BootstrapAccount, BridgeContract,
+    JstzRollup,
+};
+use nix::{
+    sys::signal::{kill, Signal},
+    unistd::Pid,
+};
+use octez::OctezThread;
+use regex::Regex;
 use std::{
     env,
-    fs::{self, File},
+    fs::{self, File, OpenOptions},
+    io::{BufRead, BufReader, Seek, Write},
     path::{Path, PathBuf},
-    process::Child,
-    thread::sleep,
+    process::{Child, Command, Stdio},
+    thread::{self, sleep},
     time::Duration,
 };
 
 use console::style;
-use jstz_rollup::{
-    deploy_ctez_contract, rollup::make_installer, BootstrapAccount, BridgeContract,
-    JstzRollup,
-};
-use log::{debug, info};
-use octez::OctezThread;
+use log::info;
 use prettytable::{format::consts::FORMAT_DEFAULT, Cell, Row, Table};
 use tempfile::TempDir;
 use tokio::task;
 
+macro_rules! debug {
+    ($file:expr, $($arg:tt)*) => {
+        writeln!($file, $($arg)*).expect("Failed to write to log file");
+        $file.flush().expect("Failed to flush log file");
+    };
+}
+
```

Comment:
I think it is possible to avoid this hacky debug macro by using `env_logger` to configure the output to a file

Reply:
Added ticket to fix in the future